code-samples/additional-features/cooldowns/commands/moderation/kick.js

```diff
@@ -0,0 +1,13 @@
+const { SlashCommandBuilder } = require('discord.js');
+
+module.exports = {
+	cooldown: 5,
+	data: new SlashCommandBuilder()
+		.setName('kick')
+		.setDescription('Select a member and kick them (but not really).')
+		.addUserOption(option => option.setName('target').setDescription('The member to kick')),
```

Comment:
```suggestion
		.addUserOption(option => option.setName('target').setDescription('The member to kick').setRequired(true)),
```

Make it required or account for it being null, possibly

Reply:
for this use case, the former, definitely